pkg/cmd/populate/populate_test.go

```diff
@@ -189,7 +189,7 @@ func TestPopulate(t *testing.T) {
 		assertionFunc       func(t *testing.T, fakeStore *secretstorefake.FakeSecretStore, mavenSettings string)
 	}
 	gcpLocation := "123456"
-	vaultLocation := ""
+	vaultLocation := "https://127.0.0.1:8200"
```

Comment:
unit tests probably failing around this line. Works locally but not sure what environment the remote tests are running under. Will check up on this later